spec/presenters/sufia/trophy_presenter_spec.rb

```diff
@@ -16,7 +16,7 @@
   end
 
   let(:presenter) { described_class.new(solr_document) }
-  let(:solr_document) { SolrDocument.new(id: '123456', has_model_ssim: 'GenericWork') }
+  let(:solr_document) { SolrDocument.new(id: '123456', has_model_ssim: 'GenericWork', title_tesim: 'A Title') }
```

Comment:
@mjgiarlo yes, but in the end it doesn't matter since it's cast to an array anyway.